template.yaml

```diff
@@ -204,6 +204,9 @@ Resources:
         Types:
           - !Ref ApiGatewayEndpointType
 
+  CristinImportNviQueue:
+    Type: AWS::SQS::Queue
+
```

Comment:
Should we add some comments that these resources should be deleted once cristin-import have been done in prod?